addons/sale_timesheet/models/project.py

```diff
@@ -249,25 +298,52 @@ def get_panel_data(self):
         panel_data = super(Project, self).get_panel_data()
         return {
             **panel_data,
+            'analytic_account_id': self.analytic_account_id.id,
+            'sold_items': self._get_sold_items(),
             'profitability_items': self._get_profitability_items()
         }
 
+    def _get_sale_order_lines(self):
+        sale_orders = self.sale_order_id | self.tasks.sale_order_id
+        return self.env['sale.order.line'].search([('order_id', 'in', sale_orders.ids), ('is_service', '=', True)], order='id asc')
+
+    def _get_sold_items(self):
+        sols = self._get_sale_order_lines()
+        number_sale_orders = len(self.sale_order_id | self.tasks.sale_order_id)
```

Comment:
I think it is equivalent to: 
```suggestion
        number_sale_orders = len(sols.order_id)
```
isn't it ? 🙂